README.md

```diff
@@ -1,2 +1,162 @@
+
+https://rdzaman20000.github.io/GamersPortal/
+
+## 🛠 Built With <a name="built-with">HTML, CSS and JS</a>
+
+### Tech Stack <a name="tech-stack">N/A</a>
+
+HTML,CSS and JS is used
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+A simple webpage with Home and About page. And there is an online version of this webpage.
+
+- **Button_animation**
+- **links**
+- **Design**
+- **Contact_Form**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo <a name="live-demo"></a>
+
+The live demo of my project will be published as soon as its merged with the main branch . Keep tuned for the live demo. Thanks.
+
+- **Button_animation**
+- **links**
+- **Design**
+- **Contact_Form**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+A new developer can use the template from this project
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have NodeJS installed
+- A Web Browser Google Chrome, Firefox, etc
+- A Code Edit
+- or VSCode, Atom etc 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  git clone git@github.com:Rdzaman20000/GamersPortal.git
+
+### Install
+
+Install this project with:
+
+Live server extension in visual studio code
+
+### Usage
+
+To run the project, execute the following command:
+
+Right click the html file and select the option "Open with live server";
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@Rdzaman20000](https://github.com/Rdzaman20000)
+- Twitter: [@99_spectre](https://twitter.com/99_spectre)
+- LinkedIn: [@md-rafiduzzaman-nahin](https://www.linkedin.com/in/md-rafiduzzaman-nahin-7431ab1b4/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+I will add  database connection
+
+- [ ] **User_interface**
+- [ ] **Database_connection**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Rdzaman20000/GamersPortal/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Let me know if u liked this portfolio project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Cindy Shin as the original design was created by her. 
```

Comment:
- [ ] [OPTIONAL] To give her full credit as suggested, I think you should use the resources the previous code reviewer gave you [here](https://github.com/Rdzaman20000/GamersPortal/pull/1#discussion_r1256716814). It includes several links accrediting her identity and a sample of works. This shows your openness and honesty.